src/aten/EmptyTensor.cpp

```diff
@@ -1,4 +1,5 @@
 #define TORCH_ASSERT_NO_OPERATORS
+#include "EmptyTensor.h"
```

Comment:
Why add it?

Reply:
It's the interface of xpu empty_strided.

Reply:
Before adding it, we can pass compilation. Why add it?

Reply:
Ok, you mean the decorator declared in header need to be included during cpp compilation to let compiler know the attribute.

Reply:
Can we use `aten/EmptyTensor.h` here?

Reply:
if you mean `#include<aten/EmptyTensor.h>` , yes.